frontend/src/pages/ConnectionsPage/ConnectionsPage.tsx

```diff
@@ -4,6 +4,7 @@ import { ConnectionsList } from '../../components/ConnectionsList'
 import { ApplicationState } from '../../store'
 import { useSelector } from 'react-redux'
 import analytics from '../../helpers/Analytics'
+import { emit } from '../../services/Controller'
```

Comment:
This should be removed. Network interfaces don't have anything to do with the connections page.

Reply:
I kept this one. I'd like to have it on the connections page, devices page and settings page. So we have a better chance of it getting updated when a user comes back to the app. If you have another idea for this let me know.

Reply:
Please see final review comment for idea of heartbeat hook instead.